atmo/settings.py

```diff
@@ -114,6 +120,21 @@
     )
 }
 
+REDIS_URL = config(
+    'REDIS_URL',
+    cast=parse.urlparse
+)
+
+RQ_QUEUES = {
+    'default': {
+        'HOST': REDIS_URL.hostname,
+        'PORT': REDIS_URL.port,
+        'DB': 0,
+        'PASSWORD': REDIS_URL.password,
+        'DEFAULT_TIMEOUT': 360,
```

Comment:
That means jobs will be considered lost if they don't finish in 6 minutes, do you think that's okay? Should we increase to 10 mins?